storage/postgres/tree_storage.go

```diff
@@ -141,6 +145,13 @@ func (p *pgTreeStorage) getStmt(ctx context.Context, skeleton *statementSkeleton
 	}
 
 	statement, err := expandPlaceholderSQL(skeleton)
+
+	var counter = skeleton.restPlaceholders*skeleton.num + 1
+	for strings.Contains(statement, "<param>") {
+		statement = strings.Replace(statement, "<param>", "$"+strconv.Itoa(counter), 1)
+		counter++
+	}
```

Comment:
Isn't this reinventing `expandPlaceholderSQL()`?

Reply:
Expand placeholder wasn't making sense to me for these extra values.  As far as I can/could see, expand placeholders didn't handle single replacement values.  I can look at it later, or you can show me how to use it more effectively.